src/test/fixtures/config/forms/local/cover-letter-extension.json

```diff
@@ -0,0 +1,76 @@
+{
+    "name": "Covering Letter (extension)",
```

Comment:
needs to have some formatter, indentation issue

Reply:
done